ui/app/controllers/fusor_ui/concerns/application_controller_extensions.rb

```diff
@@ -0,0 +1,14 @@
+module FusorUi
+  module Concerns
+    module ApplicationControllerExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        def welcome
+          redirect_to new_fusor_deployment_path
+        end
```

Comment:
Fascinating. For my own edification, how does this work? When I did it before as a plugin I could not get the welcome page to not run. I had my own ```ApplicationControllerExtensions``` 

```
module ForemanPasswordPrompt
  module Concerns
    module ApplicationControllerExtensions
      extend ActiveSupport::Concern

      included do
        before_filter :change_password
      end

      def change_password
        if User.current && (User.current.created_at >= User.current.updated_at)
          redirect_to change_password_path
        end
      end
    end
  end
end
```

Clearly the magic has something to do with ```def welcome```. Are you basically trapping the welcome url before it hits welcome?